packages/broadway.rb

```diff
@@ -15,36 +15,32 @@ class Broadway < Package
   depends_on 'gtk3'
 
   def self.build
-    system "echo 'GDK_BACKEND=broadway' > .broadway.env"
-    system "echo 'XDG_RUNTIME_DIR=/var/run/chrome' >> .broadway.env"
-    system "echo 'BROADWAY_DISPLAY=:5' >> .broadway.env"
+    system "echo 'export GDK_BACKEND=broadway' > .broadway.env"
+    system "echo 'export XDG_RUNTIME_DIR=/var/run/chrome' >> .broadway.env"
+    system "echo 'export BROADWAY_DISPLAY=:5' >> .broadway.env"
     system "echo '#!/bin/bash' > initbroadway"
+    system "echo >> initbroadway"
     system "echo 'source ~/.broadway.env' >> initbroadway"
+    system "echo >> initbroadway"
     system "echo 'BROADWAYD=\$(pidof broadwayd 2>/dev/null)' >> initbroadway"
     system "echo 'if [ -z \"\${BROADWAYD}\" ]; then' >> initbroadway"
     system "echo '  [ -f #{CREW_PREFIX}/bin/stopsommelier ] && stopsommelier' >> initbroadway"
-    system "echo '  broadwayd \${BROADWAY_DISPLAY} &' >> initbroadway"
+    system "echo '  broadwayd \${BROADWAY_DISPLAY} &>/dev/null &' >> initbroadway"
```

Comment:
Don't we want to display errors if they exist?  No harm in not including this.

Reply:
> Silences broadway and sommelier daemons (for certain cases in which output from ~/.bashrc is undesirable)

Reply:
Are you saying any errors that might happen are undesirable also?

Reply:
Not necessarily. I just don't want to interfere with the users' ability to use `scp` and `rcp`.

Reply:
`broadwayd process 4126 is running` is the only thing that it displays, and then it exits.